lighthouse-cli/test/smokehouse/lighthouse-runners/bundle.js

```diff
@@ -30,6 +30,32 @@ global.require = originalRequire;
 // @ts-expect-error - not worth giving test global an actual type.
 const lighthouse = global.runBundledLighthouse;
 
+/**
+ * Clean the config of any additional audits that might not be able to be referenced.
+ * A new audit is detected by finding any audit entry with a path and no options.
+ *
+ * @param {LH.Config.Json=} configJson
+ */
+function cleanConfig(configJson) {
+  if (!configJson) return;
+
+  if (configJson.audits) {
+    configJson.audits = configJson.audits.filter(audit => {
+      // Just a string, is adding a new audit that won't be in the bundle, prune it.
+      if (typeof audit === 'string') return false;
+      // If it's the audit implementation directly, it's a new audit, prune it. (though not currently possible)
+      if (typeof audit !== 'object') return false;
+      // Keep the audit only if it is setting options, which is the signal that it's an existing core audit.
+      return Boolean(audit.options);
+    });
+  }
+
+  if (configJson.categories) {
```

Comment:
what's going on here? does this not break `form-config` / form smoke?

Reply:
form-config is disabled because it doesn't work without the flag :) 

Other options I see:

- hardcode this function to exclude only this oopif audit
- Add this as a default audit
- Include this audit in the DT bundle
- Test a different bundle than the DT bundle that does include this audit

Any preferences?

Reply:
ha, sorry, missed that you had already replied